src/functions.test.js

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line
+import { add, removeTask } from './functions';
+
+global.TextEncoder = require('util').TextEncoder;
+global.TextDecoder = require('util').TextDecoder;
+
+// eslint-disable-next-line
+const { JSDOM } = require('jsdom');
+
+const tasksArray = [];
+
+test('stringLength returns the correct length of a string', () => {
```

Comment:
- [ ] Please make sure to use the `describe` keyword in each of your tests to identify them. See the requirements [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_1.md#:~:text=Make%20sure%20you%20group%20your%20tests%20using%20the%20describe()%20method.).